api/api.proto

```diff
@@ -2512,6 +2512,25 @@ service OpenStorageVolume {
           body: "*"
         };
     }
+
+
+}
+
+
+// OpenStorageWatcher is a service used to stream events in the system
+service OpenStorageWatch{
+
+    // Watch streams of event exist in the system
+    //
+    // Requires access AccessType.Read
+    rpc Watch(SdkWatchRequest)
+    returns (stream SdkWatchResponse) {
+        option(google.api.http) = {
+          post: "/v1/watch/watch"
```

Comment:
I think this should be just `/v1/watch`. We are not watching on specific resources anymore so we don't need the extra prefix of service name anymore.